.github/workflows/release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Releases
+
+on:
+  push:
+    branches: [ main ]
+
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: conventional Changelog Action
+
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3
+        with:
+          gihtub-token: ${{ secrets.CHANGELOG_RELEASE }}
+
+      - name: create release
+        uses: actions/create-release@v1
+        if: ${{ steps.changelog.skipped == 'false' }}
+
+        env:
+          GITHUB_TOKEN: ${{ secrets.CHANGELOG_RELEASE }}
```

Comment:
```suggestion
          GITHUB_TOKEN: ${{ secrets.github_token }}
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix